guava/pom.xml

```diff
@@ -248,12 +248,43 @@
     </plugins>
   </build>
   <profiles>
+    <profile>
+      <id>spdx</id>
+      <activation>
+        <jdk>11</jdk>
+      </activation>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.spdx</groupId>
+            <artifactId>spdx-maven-plugin</artifactId>
+            <version>0.7.3</version>
+            <executions>
+              <execution>
+                <id>build-spdx</id>
+                <goals>
+                  <goal>createSPDX</goal>
+                </goals>
+              </execution>
+            </executions>
+            <configuration>
+              <useArtifactID>true</useArtifactID>
+              <includeTransitiveDependencies>false</includeTransitiveDependencies>
+            </configuration>
+          </plugin>
+        </plugins>
+      </build>
+    </profile>
     <profile>
       <id>srczip-parent</id>
       <activation>
         <file>
           <exists>${java.home}/../src.zip</exists>
         </file>
+        <property>
+          <name>maven.javadoc.skip</name>
+          <value>false</value>
+        </property>
```

Comment:
The `srczip` from JDK will now only be unzipped and mounted if `-Dmaven.javadoc.skip` is equal to `false`.

Reply:
I'm concerned about this change; I added it because the `srczip` can interfere with the SPDX document. SPDX thinks the JDK sources are an actual dependency, but they aren't, they are just included for Javadoc.

Reasons why this change concerns me:

- People may expect the Javadoc to build by default as it does now; so this is a breaking change from a DX perspective
- CI may rely on Javadoc building by default

I want to roll it back, but I'm not sure how yet. I may eliminate SPDX completely, or I may try to find another way around the `srczip` problem. Ultimately the JDK sources are _not_ a "dependency" so if there is a way to model them without relying on that, maybe that could work.

Reply:
I dug up where I'd mentioned this previously (https://github.com/google/guava/issues/7065#issuecomment-1974119982), but just to reiterate: Inheriting JDK Javadoc hasn't been working in a while now, at least in the configurations that we've been using. Feel free to just rip it out rather than try to keep it working in the configurations that it does work for (if any) or try to fix it. (I believe that's the only reason that we mess with `src.zip`.)

Reply:
Nice, okay, thank you. I'd missed that.

Reply:
Update: JDK srczip removal has been split out into #7092